### PR TITLE
Add readme for project

### LINKED ISSUE #5: Add README
Description, features, maybe setup instructions.

### PR DESCRIPTION
**Pull request by Satont**
fixes #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Gallery (pics-sharer)
```
Comment (AlexWayfer) [inline README.md:1]: 1. Not sure about "sharer" word.
2. It's a title! Just left it original, description is below.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+![](https://img.shields.io/github/workflow/status/satont/gallery/Publish%20Docker/master?label=docker&style=for-the-badge)
+
+---
+# Description
+
+Personally, i getting tired saving pictures to PC from my friends. So, i get simple idea, how i can store and share images from my friends/me.
```
Comment (AlexWayfer) [inline README.md:8]:
```suggestion
Personally, I'm tired of saving pictures from my friends to my PC. So, I get a simple idea how I can store and share images from my friends and/or me.
```

**Comment by Satont** [comment]
Hi, @AlexWayfer.

Thank you for your suggestions.

**Comment by Satont** [comment]
@AlexWayfer is it all? Or you can add something else?

**Comment by AlexWayfer** [comment]
> @AlexWayfer is it all? Or you can add something else?

I wanted to rewrite (improve) "How it works" instructions, include a "connect this project to your Discord channel", describe a channel somehow like "common" or "generating person-specific channels", but I don't understand mechanics completely and it's pretty hard for me. Without it that's all.

**Comment by Satont** [comment]
> > @AlexWayfer is it all? Or you can add something else?
> 
> I wanted to rewrite (improve) "How it works" instructions, include a "connect this project to your Discord channel", describe a channel somehow like "common" or "generating person-specific channels", but I don't understand mechanics completely and it's pretty hard for me. Without it that's all.

I have own discord server, where you can test it. Do want invite link?

**Pull request merged**